src/components/TimelineNav/TimelineNav.stories.tsx

```diff
@@ -110,13 +110,13 @@ export default {
               nisi ut aliquip ex{' '}
             </p>
           </Text>
-        </ListDetailPanel>
+        </TimelineNavPanel>
       </>
     ),
   },
 } as Meta;
 
-type Args = React.ComponentProps<typeof ListDetail>;
+type Args = React.ComponentProps<typeof TimelineNav>;
 
 export const Default: StoryObj<Args> = {
   parameters: {
```

Comment:
Does this axe rule still need to be disabled?

Reply:
Nope. Thanks!